ci-operator/step-registry/rosa/cluster/upgrade/hcp/rosa-cluster-upgrade-hcp-commands.sh

```
@@ -217,9 +185,15 @@ if [[ "$CHANNEL_GROUP" == "nightly" ]]; then
   exit 1
 fi
 
-HOSTED_CP=$(rosa describe cluster -c $cluster_id -o json | jq -r '.hypershift.enabled')
 init_version=$(rosa describe cluster -c $cluster_id -o json | jq -r '.openshift_version')
 
+UPGRADE_CHANNEL=${UPGRADE_CHANNEL:-}
+if [[ -n "${UPGRADE_CHANNEL}" ]]; then
+  log "Changing cluster channel to ${UPGRADE_CHANNEL}"
+  rosa edit cluster -c $cluster_id --channel "${UPGRADE_CHANNEL}"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)